proto/cosmos/group/v1beta1/types.proto

```diff
@@ -174,98 +171,101 @@ message Proposal {
-  // execution messages must be before this end time to be included in the election. After the timeout timestamp the
-  // proposal can not be executed anymore and should be considered pending delete.
-  google.protobuf.Timestamp timeout = 11 [(gogoproto.nullable) = false, (gogoproto.stdtime) = true];
+  // Until a final tally has happened the status is unfinalized
+  PROPOSAL_RESULT_UNFINALIZED = 1;
 
-  // ExecutorResult defines types of proposal executor results.
-  enum ExecutorResult {
-    option (gogoproto.goproto_enum_prefix) = false;
+  // Final result of the tally
+  PROPOSAL_RESULT_ACCEPTED = 2;
 
-    // An empty value is not allowed.
-    EXECUTOR_RESULT_UNSPECIFIED = 0 [(gogoproto.enumvalue_customname) = "ProposalExecutorResultInvalid"];
+  // Final result of the tally
+  PROPOSAL_RESULT_REJECTED = 3;
+}
 
-    // We have not yet run the executor.
-    EXECUTOR_RESULT_NOT_RUN = 1 [(gogoproto.enumvalue_customname) = "ProposalExecutorResultNotRun"];
+// ProposalExecutorResult defines types of proposal executor results.
+enum ProposalExecutorResult {
+  option (gogoproto.goproto_enum_prefix) = false;
 
-    // The executor was successful and proposed action updated state.
-    EXECUTOR_RESULT_SUCCESS = 2 [(gogoproto.enumvalue_customname) = "ProposalExecutorResultSuccess"];
+  // An empty value is not allowed.
+  PROPOSAL_EXECUTOR_RESULT_UNSPECIFIED = 0;
 
-    // The executor returned an error and proposed action didn't update state.
-    EXECUTOR_RESULT_FAILURE = 3 [(gogoproto.enumvalue_customname) = "ProposalExecutorResultFailure"];
-  }
+  // We have not yet run the executor.
+  PROPOSAL_EXECUTOR_RESULT_NOT_RUN = 1;
 
-  // executor_result is the final result based on the votes and election rule. Initial value is NotRun.
-  ExecutorResult executor_result = 12;
+  // The executor was successful and proposed action updated state.
+  PROPOSAL_EXECUTOR_RESULT_SUCCESS = 2;
 
-  // msgs is a list of Msgs that will be executed if the proposal passes.
-  repeated google.protobuf.Any msgs = 13;
+  // The executor returned an error and proposed action didn't update state.
+  PROPOSAL_EXECUTOR_RESULT_FAILURE = 3;
 }
 
-// Tally represents the sum of weighted votes.
-message Tally {
+// TallyResult represents the sum of weighted votes for each vote option.
+message TallyResult {
   option (gogoproto.goproto_getters) = false;
 
-  // yes_count is the weighted sum of yes votes.
-  string yes_count = 1;
+  // yes is the weighted sum of yes votes.
+  string yes = 1;
 
-  // no_count is the weighted sum of no votes.
-  string no_count = 2;
+  // abstain is the weighted sum of abstainers.
+  string abstain = 2;
 
-  // abstain_count is the weighted sum of abstainers
-  string abstain_count = 3;
+  // no is the weighted sum of no votes.
+  string no = 3;
 
-  // veto_count is the weighted sum of vetoes.
-  string veto_count = 4;
+  // no_with_veto is the weighted sum of veto.
+  string no_with_veto = 4;
 }
```

Comment:
quick poll: what do we think is more intuitive: "Yes" or "YesCount" etc... ?

Reply:
Hmm, the fact that it's a string can be confusing, I prefer `_count` so that we know it represents a number. Will change in gov too.